eo-runtime/src/test/java/org/eolang/MainTest.java

```diff
@@ -84,6 +84,14 @@ public void jvmFullRunWithError() throws Exception {
         );
     }
 
+    @Test
+    public void objectNotFoundException() throws Exception {
+        MatcherAssert.assertThat(
+            MainTest.exec("--verbose", "unavailable-name"),
```

Comment:
@levBagryansky should we use "--verbose" flag to print "Can not find \"unavailable-name\" object"?  Or the compiler will print the message anyway?

Reply:
@volodya-lombrozo Indeed, it is printed anyway